api/project.json

```diff
@@ -1,12 +1,22 @@
 {
-  "root": "api",
   "targets": {
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "api",
+        "command": "gulp build"
+      },
+      "inputs": ["{projectRoot}/**/*", "!{projectRoot}/_build/**/*"],
```

Comment:
@miguel-lansdorf Please check if there are any environment variables that can impact the api build result.

Reply:
No env variables are required for the build